app/js/services.js

```diff
@@ -0,0 +1,57 @@
+floripaApp.factory('searchService', function () {
+    return {};
+});
+
+floripaApp.factory('FindRoutesByStopName', ['$resource', '$http',
+
+  function ($resource, $http) {
+
+        $http.defaults.headers.common['Authorization'] = authString;
+
+        return $resource(api_url + 'findRoutesByStopName/run/', {}, {
+            save: {
+                method: 'POST',
+                headers: {
+                    'X-AppGlu-Environment': 'staging',
+                    'Content-type': 'application/json'
+
+                }
+            }
+        });
+}]);
+
+floripaApp.factory('FindStopsByRouteId', ['$resource', '$http',
```

Comment:
My bad, this was a really terrible naming. I would refactor them into a single service and call it timetableSvc.